os/arch/arm/include/amebalite/chip.h

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Change year to 2022 for this and other files

Reply:
I have change all license year to 2022